dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/helper/PartitionHelper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.backend.helper;
+
+public class PartitionHelper {
+
+    private static final String PARTITION_SEPARATOR = "|";
+
+    private PartitionHelper() {
+        // empty
+    }
+
+    public static String partition(String modelId, String id) {
+        return id + PARTITION_SEPARATOR + " RuntimeModel=" + modelId;
+    }
+
+    public static String removePartition(String id) {
+        if (id != null) {
+            int sepatorIndex = id.indexOf(PARTITION_SEPARATOR);
+            if (sepatorIndex > -1) {
+                return id.substring(0, sepatorIndex);
+            }
+        }
+        return id;
```

Comment:
Does it make sense to return `null` in case `id==null`?

Reply:
Yes, I mean, this method should either remove the partition information or the original ID if there's no partition - the mark for partition in our case is the "|" char - if the ID has no partition char, then the original ID is returned - even if it is nulll